src/EmailObject.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+module.exports = class EmailObject {
+  constructor(emailParams) {
+    this.email = {
+      from: {
+        email: emailParams.from,
+        name: emailParams.fromName,
+      },
+      to: emailParams.recipients,
+      cc: emailParams.cc,
+      bcc: emailParams.bcc,
+      reply_to: emailParams.reply_to,
```

Comment:
this should be:
`reply_to: emailParams.replyTo`

Reply:
Ooops. That's a good catch, thanks :)